package.json

```diff
@@ -27,5 +27,8 @@
     "prettier": "^2.8.8",
     "ts-jest": "^29.1.0",
     "typescript": "^5.0.4"
+  },
+  "dependencies": {
+    "@web/test-runner-playwright": "^0.10.0"
```

Comment:
convert to dev-dep

Reply:
Looks like this is required unfortunately. This is pretty gross and we want to avoid users from having to install.